shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java

```diff
@@ -134,22 +135,24 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
     shellState.setTableName(tableName); // switch shell to new table context
 
     if (cl.hasOption(createTableNoDefaultIters.getOpt())) {
-      for (String key : IteratorConfigUtil.generateInitialTableProperties(true).keySet()) {
-        shellState.getAccumuloClient().tableOperations().removeProperty(tableName, key);
-      }
+      Set<String> initialProps = IteratorConfigUtil.generateInitialTableProperties(true).keySet();
+      shellState.getAccumuloClient().tableOperations().modifyProperties(tableName,
+          properties -> initialProps.forEach(properties::remove));
     }
 
     // Copy options if flag was set
     if (cl.hasOption(createTableOptCopyConfig.getOpt())) {
       if (shellState.getAccumuloClient().tableOperations().exists(tableName)) {
         final Map<String,String> configuration = shellState.getAccumuloClient().tableOperations()
             .getConfiguration(cl.getOptionValue(createTableOptCopyConfig.getOpt()));
-        for (Entry<String,String> entry : configuration.entrySet()) {
-          if (Property.isValidTablePropertyKey(entry.getKey())) {
-            shellState.getAccumuloClient().tableOperations().setProperty(tableName, entry.getKey(),
-                entry.getValue());
-          }
-        }
+
+        Map<String,
+            String> propsToAdd = configuration.entrySet().stream()
+                .filter(entry -> Property.isValidTablePropertyKey(entry.getKey()))
+                .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
+
+        shellState.getAccumuloClient().tableOperations().modifyProperties(tableName,
+            properties -> properties.putAll(propsToAdd));
```

Comment:
```suggestion
        shellState.getAccumuloClient().tableOperations().modifyProperties(tableName,
            properties -> configuration.entrySet().stream()
                .filter(entry -> Property.isValidTablePropertyKey(entry.getKey()))
                .forEach(entry -> properties.put(entry.getKey(), entry.getValue())));
```

Reply:
This change would make it so we only need to iterate once over the properties vs having to iterate over the map twice (once to filter into a new map and then again when putAll() is called)

Reply:
That makes sense, ill include this change.

Reply:
Do you think I should refactor all of the spots that are similar to this to reduce the number of times we iterate?